### PR TITLE
Refactor to use foreman-js packages

### PR DESCRIPTION
**Pull request by laviro**

### CODE REVIEW

**Comment by laviro** [comment]
tests and lintings are working, but stories are broken - seem to be an issue of storybook itself.

**Comment by laviro** [comment]
@sharvit, could you take a look why stories are broken? thanks :pray:

**Comment by sharvit** [comment]
> @sharvit, could you take a look why stories are broken? thanks

@laviro, can you please try to reinstall again?
I had the same issue in core and somehow it works now (maybe storybook had a bug and they fix it).

File: package.json [hunk 1]
```diff
@@ -20,106 +20,26 @@
...
-      "node_modules",
-      "webpack"
-    ],
-    "setupFiles": [
-      "raf/polyfill",
-      "jest-prop-type-error",
-      "./webpack/test_setup.js"
-    ]
+    "jed": "^1.1.1",
+    "react-intl": "^2.8.0"
```
Comment (sharvit) [inline package.json]: Do you actually use `jed` and `reat-intl`?

Comment (laviro) [reply package.json]: yes, using `react-intl` for relative time
and `jed` for sprintf

**Comment by laviro** [comment]
yes, tried to remove and re-install.
I am using the same storybook and storybook/addon-docs versions as Foreman core - where it works.

Still getting the following error in storybook:
```console
Uncaught TypeError: _acorn.Parser.extend is not a function
    at Object../node_modules/@storybook/addon-docs/dist/frameworks/react/lib/inspection/acornParser.js (vendors~main.431981621d293f7b6f51.bundle.js:6100)
    at __webpack_require__ (runtime~main.431981621d293f7b6f51.bundle.js:791)
    at fn (runtime~main.431981621d293f7b6f51.bundle.js:151)
    at Object../node_modules/@storybook/addon-docs/dist/frameworks/react/lib/inspection/inspectValue.js (vendors~main.431981621d293f7b6f51.bundle.js:6360)
    at __webpack_require__ (runtime~main.431981621d293f7b6f51.bundle.js:791)
    at fn (runtime~main.431981621d293f7b6f51.bundle.js:151)
    at Object../node_modules/@storybook/addon-docs/dist/frameworks/react/lib/inspection/index.js (vendors~main.431981621d293f7b6f51.bundle.js:6329)
    at __webpack_require__ (runtime~main.431981621d293f7b6f51.bundle.js:791)
    at fn (runtime~main.431981621d293f7b6f51.bundle.js:151)
    at Object../node_modules/@storybook/addon-docs/dist/frameworks/react/propTypes/createType.js (vendors~main.431981621d293f7b6f51.bundle.js:6484)
```

**Comment by laviro** [comment]
@sharvit @ShimShtein, I found that foreman-tasks storybook doesn't work as well,
so maybe it is an issue with `vendor` or `@storybook/addon-docs`

I think we should merge this PR to use vendor already to stay aligned with foreman/develop

**Comment by sharvit** [comment]
Thanks @laviro, opened an issue in foreman-js: https://github.com/theforeman/foreman-js/issues/122

**Comment by laviro** [comment]
Thanks @ShimShtein, 
updated `engine.rb` with `requires_foreman '> 1.24'`

**Comment by ShimShtein** [comment]
Thanks @laviro!

**Pull request merged**